tests/neo.UnitTests/SmartContract/UT_InteropService.cs

```diff
@@ -861,6 +861,16 @@ public void TestContract_Destroy()
 
         }
 
+        [TestMethod]
+        public void TestContract_CreateStandardAccount()
+        {
```

Comment:
@Qiao-Jin Could you add some UT with wrong public keys?

Reply:
> @Qiao-Jin Could you add some UT with wrong public keys?

Sure, I added some cases where invalid public keys set off different kinds of exceptions. Should there be any cases omitted please inform me:)